egs2/minds14/asr1/run_new.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```

Comment:
Please keep one `run.sh`

Reply:
Thanks for the feedback!
We are still running some experiments at the moment which is why we had multiple run.sh files.
We'll make sure to leave only one once the experiments are done.